go/oauth2util/main/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/google/oauth2l/go/oauth2util"
+	"golang.org/x/oauth2"
+)
+
+const (
+	// Common prefix for google oauth scope
+	scopePrefix = "https://www.googleapis.com/auth/"
+)
+
+func help() {
+	fmt.Println("Usage: oauth2l --json <secret.json> " +
```

Comment:
There is no max col limit in Go.

fmt.Println("Usage: auth2l -json <secret.json> {fetch|header|token} scope1 scope2 ...")

is so much easier to read without having to worry what concat is doing.